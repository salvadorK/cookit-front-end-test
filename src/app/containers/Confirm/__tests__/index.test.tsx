import * as React from 'react';
import { Confirm } from '..';
import { createRenderer } from 'react-test-renderer/shallow';

import * as redux from 'react-redux';

const spy = jest.spyOn(redux, 'useSelector');
spy.mockReturnValue({ email: '', postalCode: '' });

const shallowRenderer = createRenderer();

describe('<Confirm />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<Confirm />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
