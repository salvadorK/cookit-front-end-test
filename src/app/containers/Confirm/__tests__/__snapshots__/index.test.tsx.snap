// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Confirm /> should render and match the snapshot 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title />
    <meta
      content=""
      name="description"
    />
  </Helmet>
  <NavBar />
  <Confirm__Wrapper>
    <h1>
      Confirm?
    </h1>
    <h2>
      
    </h2>
    <h2>
      
    </h2>
    <Confirm__Wrapper>
      <div>
        <Confirm__ConfirmButton
          onClick={[Function]}
          type="submit"
        >
          Back
        </Confirm__ConfirmButton>
        <Confirm__ConfirmButton
          onClick={[Function]}
          type="submit"
        >
          OK
        </Confirm__ConfirmButton>
      </div>
    </Confirm__Wrapper>
  </Confirm__Wrapper>
</React.Fragment>
`;
